clients/ui/frontend/src/app/pages/settings/roleBinding/RoleBindingPermissionsTableRow.tsx

```diff
@@ -69,7 +69,6 @@ const RoleBindingPermissionsTableRow: React.FC<RoleBindingPermissionsTableRowPro
   onEdit,
   onDelete,
 }) => {
-  // TODO: We don't have project context yet and need to add logic to show projects permission tab under manage permissions of MR - might need to move the project-context part to shared library
   const currentUser = useUser();
```

Comment:
we need to keep this comment until we have project-context. Could you please revert this back?